src/main/java/ru/itmo/sd/deadliner2bot/dto/TodoDto.java

```diff
@@ -0,0 +1,18 @@
+package ru.itmo.sd.deadliner2bot.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.annotation.Nullable;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class TodoDto {
```

Comment:
Why can't we use Todo class? The same for TodoNotification

Reply:
Model classes have a complicated lifetime managed by JPA. We need something persistent instead.

Reply:
Moreover, frequently we don't require fields those reference other entities, so it's better to store data in more lightweight DTOs.

Reply:
> Model classes have a complicated lifetime managed by JPA. We need something persistent instead.

I'm still not understanding, why we need them. Can you clarify please, what are the problems with lifetime?

> Moreover, frequently we don't require fields those reference other entities, so it's better to store data in more lightweight DTOs.

2 instances worse than 1, so if we can escape additional objects, we should do it.

Reply:
There is no point to store in long-term instances (like `Runnables` for `SchedulingExecutorService`) data with unnecessary fields. 
It is a common practice to use DTOs alongside entities:
https://www.baeldung.com/java-dto-pattern
https://www.baeldung.com/entity-to-and-from-dto-for-a-java-spring-application

Reply:
I've read. As for me, this is not our case. DTO's are used to transfer data to other serivices - in such case amount of data is important. We are not transfering, no serializing or JSON messages, only using by runnables. I recommend to avoid these classes for now. Also, do you use TodoNotificationDto?